### PR TITLE
tidb-in-kubernetes, dev: organize and add documents to TOC

### PR DESCRIPTION
**Pull request by lilin90**
### What is changed, added or deleted? 

- Rename several files about TiDB in K8s `get-started`
- Put TiDB in K8s related documents to new directories and change the originally inappropriate directory
- Add TiDB in K8s related documents to TOC 
- Take "upgrade TiDB Operator" out of the original file
- Fix typos and update wording

### What is the related PR or file link(s)? 

N/A

### Which version does your change affect? 

- dev
- v3.0

### Checklist 

- [x] Add a new file to `TOC.md`
- [x] No Tab spaces anywhere 
- [x] Leave a blank line both before and after a code block
- [x] Keep the first level heading consistent with `title` in metadata
- [x] Use *four* spaces for each level of indentation except that in `TOC.md`

### CODE REVIEW

**Comment by lilin90** [comment]
@dcalvin @tennix FYI. When the content is ready, I'll also make the same changes to `v3.0`.

File: dev/how-to/maintain/tidb-in-kubernetes/tidb-cluster.md [hunk 1]
```diff
@@ -259,7 +259,7 @@ TiDB 通过 Prometheus 和 Grafana 监控 TiDB 集群。TiDB 集群创建时，
 kubectl port-forward -n ${namespace} svc/${releaseName}-grafana 3000:3000 &>/tmp/portforward-grafana.log
 ```
 
-然后在浏览器中打开 http://localhost:3000，默认用户名和密码都为 `admin`。
+然后在浏览器中打开 [http://localhost:3000](http://localhost:3000)，默认用户名和密码都为 `admin`。
 
 Grafana 服务默认通过 `NodePort` 暴露，如果 Kubernetes 集群支持负载均衡器，你可以修改为 `LoadBalancer`，然后通过负载均衡器访问面板。
```
Comment (tennix) [inline dev/how-to/maintain/tidb-in-kubernetes/tidb-cluster.md]: Link to `Exposing service` document.

File: dev/how-to/maintain/tidb-in-kubernetes/auto-failover.md [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
 category: how-to
 ---
 
-# 故障自动转移
+# Kubernetes 上的 TiDB 集群故障自动转移
 
 故障自动转移是指在 TiDB 集群的某些节点出现故障时，TiDB Operator 会自动添加一个节点，保证 TiDB 集群的高可用，类似于 K8s 的 Deployment 行为。
 
-由于 TiDB Opererator 基于 StatefulSet 来管理 Pod，但 StatefulSet 在某些 Pod 发生故障时不会自动创建新节点来替换旧节点，所以，TiDB Operator 扩展了 StatefulSet 的这种行为，添加了 failover 功能。
+由于 TiDB Operator 基于 StatefulSet 来管理 Pod，但 StatefulSet 在某些 Pod 发生故障时不会自动创建新节点来替换旧节点，所以，TiDB Operator 扩展了 StatefulSet 的这种行为，添加了 failover 功能。
```
[the remark this answers is not in the document]
Comment (lilin90) [reply dev/how-to/maintain/tidb-in-kubernetes/auto-failover.md]: Added for them except Pod.

File: dev/reference/configuration/tidb-in-kubernetes/backup-configuration.md [hunk 3]
```diff
@@ -5,7 +5,7 @@ category: reference
 
 # TiDB Backup 配置参数
 
-TiDB-Backup 是一个用于 TiDB 集群备份和恢复的 Helm Chart。本文详细介绍了 TiDB-Backup 的可配置参数。了解如何使用 TiDB-Backup 进行集群备份恢复，请参考[备份与恢复](/dev/how-to/maintain/backup-and-restore.md)。
+TiDB-Backup 是一个用于 TiDB 集群备份和恢复的 Helm Chart。本文详细介绍了 TiDB-Backup 的可配置参数。
```
[the remark this answers is not in the document]
Comment (lilin90) [reply dev/reference/configuration/tidb-in-kubernetes/backup-configuration.md]: Added

File: dev/how-to/maintain/tidb-in-kubernetes/auto-failover.md [hunk 4]
```diff
@@ -47,7 +47,7 @@ TiDB 集群有 PD/TiKV/TiDB 三个组件，它们各自的故障转移策略又
 
 ### TiKV 故障转移策略
 
-当一个 TiKV 节点无法正常工作后，该 TiKV 节点的状态会变为 Disconnected，30 分钟（可在部署集群时通过修改 `pd.maxStoreDownTime` 来配置）后会变成 Down 状态，TiDB Operator 会在此基础上再等待 5 分钟（tikvFailoverPeriod 可配置），如果该 TiKV 节点仍不能恢复，就会新起一个 TiKV 节点。待挂掉的 TiKV 节点恢复后，TiDB Operator 不会自动删除新起的节点，用户需要通过手工减少 TiKV 节点，恢复成原来的节点数，操作方法是将该 TiKV 节点从 `TidbCluster` 对象的 `status.tikv.failureStores` 字段中删除：
+当一个 TiKV 节点无法正常工作后，该 TiKV 节点的状态会变为 Disconnected，30 分钟（可在部署集群时通过修改 `pd.maxStoreDownTime` 来配置）后会变成 Down 状态，TiDB Operator 会在此基础上再等待 5 分钟（`tikvFailoverPeriod` 可配置），如果该 TiKV 节点仍不能恢复，就会新起一个 TiKV 节点。待挂掉的 TiKV 节点恢复后，TiDB Operator 不会自动删除新起的节点，用户需要通过手工减少 TiKV 节点，恢复成原来的节点数，操作方法是将该 TiKV 节点从 `TidbCluster` 对象的 `status.tikv.failureStores` 字段中删除：
 
 {{< copyable "shell-regular" >}}
```
[the remark this answers is not in the document]
Comment (lilin90) [reply dev/how-to/maintain/tidb-in-kubernetes/auto-failover.md]: Got it. Removed.

**Comment by lilin90** [comment]
I'll make the same changes to `v3.0` and then this PR can be merged.

**Comment by lilin90** [comment]
@dcalvin PTAL~ Thanks!

File: dev/TOC.md [hunk 5]
```diff
@@ -224,6 +234,10 @@
     + tikv-server
       - [配置参数](reference/configuration/tikv-server/configuration.md)
       - [配置文件描述](reference/configuration/tikv-server/configuration-file.md)
+    + Kubernetes 上的 TiDB 配置
```
Comment (dcalvin) [inline dev/TOC.md]:
```suggestion
    + Kubernetes 上的 TiDB
```

File: dev/TOC.md [hunk 6]
```diff
@@ -224,6 +234,10 @@
...
+      - [TiDB 集群配置](reference/configuration/tidb-in-kubernetes/cluster-configuration.md)
+      - [备份配置参数](reference/configuration/tidb-in-kubernetes/backup-configuration.md)
```
Comment (dcalvin) [inline dev/TOC.md]:
```suggestion
      - [备份配置](reference/configuration/tidb-in-kubernetes/backup-configuration.md)
```

File: v3.0/how-to/maintain/tidb-in-kubernetes/auto-failover.md [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+category: how-to
+---
+
+# Kubernetes 上的 TiDB 集群故障自动转移
+
+故障自动转移是指在 TiDB 集群的某些节点出现故障时，TiDB Operator 会自动添加一个节点，保证 TiDB 集群的高可用，类似于 K8s 的 `Deployment` 行为。
+
+由于 TiDB Operator 基于 `StatefulSet` 来管理 Pod，但 `StatefulSet` 在某些 Pod 发生故障时不会自动创建新节点来替换旧节点，所以，TiDB Operator 扩展了 `StatefulSet` 的这种行为，添加了 failover 功能。
+
+Auto Failover 功能在 TiDB Operator 中默认关闭，需要在部署 TiDB Operator 时开启，开启方法为设置 `charts/tidb-operator/values.yaml` 文件的 `controllerManager.autoFailover` 为 `true`：
```
Comment (dcalvin) [inline v3.0/how-to/maintain/tidb-in-kubernetes/auto-failover.md]: ditto

File: v3.0/TOC.md [hunk 8]
```diff
@@ -224,6 +234,10 @@
     + tikv-server
       - [配置参数](reference/configuration/tikv-server/configuration.md)
       - [配置文件描述](reference/configuration/tikv-server/configuration-file.md)
+    + Kubernetes 上的 TiDB 配置
```
Comment (dcalvin) [inline v3.0/TOC.md]: ditto

File: dev/how-to/deploy/orchestrated/tidb-in-kubernetes/alibaba-cloud.md [hunk 9]
```diff
@@ -1,3 +1,8 @@
+---
+title: 在阿里云上部署 TiDB Operator 和 TiDB 集群
```
Comment (dcalvin) [inline dev/how-to/deploy/orchestrated/tidb-in-kubernetes/alibaba-cloud.md]: 这个标题感觉有点长，是不是可以缩短为 在阿里云上部署 TiDB

File: dev/TOC.md (hunk 6 above)
Comment (dcalvin) [reply dev/TOC.md]: To be consistent with its neighbors

File: dev/how-to/deploy/orchestrated/tidb-in-kubernetes/alibaba-cloud.md (hunk 9 above)
Comment (lilin90) [reply dev/how-to/deploy/orchestrated/tidb-in-kubernetes/alibaba-cloud.md]: Yes. The original title is a little bit too long.

**Comment by lilin90** [comment]
@dcalvin Updated according to comments. Thanks! PTAL again~

**Pull request merged**